llvm/docs/HowToSubmitABug.rst

```diff
@@ -153,6 +153,62 @@ Please run this, then file a bug with the instructions and reduced .bc file
 that bugpoint emits.  If something goes wrong with bugpoint, please submit
 the "foo.bc" file and the option that llc crashes with.
 
+LTO bugs
+---------------------------
+
+If you encounter a bug that leads to crashes in the LLVM LTO phase when using
+the `-flto` option, follow these steps to diagnose and report the issue:
+
+Compile your source file to a .bc (Bitcode) file with the following flags,
```

Comment:
quote `.bc`

flags => options.

`-fuse-ld=` is an option, not a flag (boolean option).